cmd/scaffold.go

```diff
@@ -187,6 +191,14 @@ func (fnb *FlowNodeBuilder) BaseFlags() {
 	fnb.flags.UintVar(&fnb.BaseConfig.SyncCoreConfig.MaxRequests, "sync-max-requests", defaultConfig.SyncCoreConfig.MaxRequests, "the maximum number of requests we send during each scanning period")
 
 	fnb.flags.Uint64Var(&fnb.BaseConfig.ComplianceConfig.SkipNewProposalsThreshold, "compliance-skip-proposals-threshold", defaultConfig.ComplianceConfig.SkipNewProposalsThreshold, "threshold at which new proposals are discarded rather than cached, if their height is this much above local finalized height")
+
+	// unicast stream handler rate limits
+	fnb.flags.IntVar(&fnb.BaseConfig.UnicastMessageRateLimit, "unicast-message-rate-limit", 0, "maximum number of unicast messages that a peer can send per second")
+	fnb.flags.IntVar(&fnb.BaseConfig.UnicastBandwidthRateLimit, "unicast-bandwidth-rate-limit", 0, "bandwidth size in bytes a peer is allowed to send via unicast streams per second")
+	fnb.flags.IntVar(&fnb.BaseConfig.UnicastBandwidthBurstLimit, "unicast-bandwidth-burst-limit", middleware.LargeMsgMaxUnicastMsgSize, "bandwidth size in bytes a peer is allowed to send at one time")
+	fnb.flags.IntVar(&fnb.BaseConfig.UnicastRateLimitLockoutDuration, "unicast-rate-limit-lockout-duration", 10, "the number of seconds a peer will be forced to wait before being allowed to successful reconnect to the node after being rate limited")
```

Comment:
```suggestion
	fnb.flags.DurVar(&fnb.BaseConfig.UnicastRateLimitLockoutDuration, "unicast-rate-limit-lockout-duration", 10, "the number of seconds a peer will be forced to wait before being allowed to successful reconnect to the node after being rate limited")
```

Reply:
Also, please add the default value as a constant parameter in the [default config of the node(https://github.com/onflow/flow-go/blob/6f35feb404725027f593efa63a21cb19cd3a6418/cmd/node_builder.go#L263)

Reply:
Default values https://github.com/onflow/flow-go/pull/2840/commits/737e2ab46e3f7a047bbd4011e1a8c05f168430b0